src/pkg/packaging/rpm/dotnet-hostfxr-rpm_config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "maintainer_name": "Microsoft",
+  "maintainer_email": "dotnetcore@microsoft.com",
+  
+  "package_name": "%HOSTFXR_RPM_PACKAGE_NAME%",
+  "install_root": "/usr/share/dotnet",
+  "install_doc": "/usr/share/doc/%HOSTFXR_RPM_PACKAGE_NAME%/",
+  
+  "short_description": "%HOSTFXR_BRAND_NAME% %HOSTFXR_NUGET_VERSION%",
+  "long_description": ".NET Core is a development platform that you can use to build command-line applications, microservices and modern websites. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/dotnet/core). We happily accept issues and PRs.",
+  
+  "homepage": "https://github.com/dotnet/core",
+  
+  "release":{
+        "package_version":"1.0.0.0",
+        "package_revision":"1",
+        "urgency" : "low",
+        "changelog_message" : "Versioned Hostfxr"
+  },
+  
+  "control": {
+        "priority":"standard",
+        "section":"libs",
+        "architecture":"amd64"
+  },
+  
+  "copyright": "2015 Microsoft",
+  
+  "license": {
+    "type": "MIT",
```

Comment:
@richlander can you comment on this? 

The implementation is similar to what we do .deb packages. The MIT license shows in the copyright file and when we display the rpm package information.